qml/IconSVG.qml

```diff
@@ -13,26 +13,19 @@
 // limitations under the License.
 
 import QtQuick 2.0
-import QtGraphicalEffects 1.15
 
 Image {
     id: image
     smooth: true
     visible: true
-    property color color: "transparent"
-    property bool dye: false
+    source: name ? "/resources/images/icons/" + name + "_" + color + ".svg" : ""
```

Comment:
better to use an external function.
Easier to change it in the future

Reply:
Why would you change this in the future?